tests/upgrade/selinux-relabel/task.yaml

```diff
@@ -0,0 +1,36 @@
+summary: Check that correct SELinux file contexts were assigned on upgrade
+
+# TODO: enable centos once we build the policy there too
+systems: [fedora-*]
```

Comment:
We should be able to build the policy for CentOS now.

Reply:
This looks like the only thing that's left that needs to be fixed here...

Reply:
Though since 2.36.3 is out, I should go ahead and build that and update everybody...

Reply:
Even better. Ping me when you upload a new update. I'll play with it in a VM and will post karma.

Reply:
snapd 2.36.3 is in the EPEL7 repo now.